sockeye/utils.py

```diff
@@ -615,3 +615,53 @@ def all_gather_object(obj: T) -> List[T]:
     obj_list = [None] * torch.distributed.get_world_size()  # type: List[T]
     torch.distributed.all_gather_object(obj_list, obj)
     return obj_list
+
+
+def count_seq_len(sample: str, count_type: str = 'char', replace_tokens: List = []) -> int:
```

Comment:
I think mutable types as default arguments is discouraged. Maybe make it `Optional[List]` and set the default to `None`.